packages/terra-application/src/terra-dev-site/test/application-base/ApplicationBaseTest.test.jsx

```diff
@@ -1,10 +1,10 @@
 import React, { useState, useContext } from 'react';
-import { ActiveBreakpointContext } from '../../../breakpoints';
-import ApplicationLoadingOverlay from '../../../application-loading-overlay';
-import ApplicationStatusOverlay from '../../../application-status-overlay';
-import ApplicationBase from '../../../application-base';
-import NavigationPrompt from '../../../navigation-prompt';
-import { ApplicationIntlContext } from '../../../application-intl';
+import { ActiveBreakpointContext } from 'terra-application/lib/breakpoints';
```

Comment:
preparing for when these get moved to the not yet created terra-application-docs package